Orange/data/dask.py

```diff
@@ -104,6 +104,17 @@ def _filter_values(self, filter):
     def _update_locks(self, *args, **kwargs):
         return
 
+    def ensure_copy(self):
```

Comment:
ensure_copy should be removed

Reply:
Ok, sorry, that wss probably too early. So, could you explain again why was it needed?

Reply:
Normally `ensure_copy` makes a copy of `_X`, `_Y`, `_W` and `_metas` if their `base` attributes are not `None`.
Dask arrays don't have that attribute though, so this method always makes a copy just in case. I'm not sure if there is a better way to do this or if it is even necessary...